lpa-submissions-e2e-tests/cypress.json

```diff
@@ -6,9 +6,9 @@
   "viewportHeight": 1600,
   "env": {
     "demoDelay": 0,
-    "DOCUMENT_SERVICE_BASE_URL": "http://localhost:3001/api/v1",
-    "ASSUME_LIMITED_ACCESS": false,
-    "APPEAL_REPLY_SERVICE_BASE_URL": "http://localhost:3002"
```

Comment:
note - because this ticket is having to solve the 'talking to apis' problem, this "assume limited access" hack goes away, which is nice.